order.js

```diff
@@ -33,16 +33,120 @@ Render.main = (data) => {
 
     <h3 id="proofs">Proofs</h3>
     ${Render.try('proofs', data)}
+
+    <h3 id="messages">Message History</h3>
+    <div class="section">
+      ${Render.try('messages', data)}
+    </div>
   `;
   return result;
 };
 
+Render.messages = (data) => {//<small style="top: 0;" class="datetime">${item.created}</small>
+  var result = '';
+  data.order_messages.forEach((item) => {
+    var msgDate = new Date(item.created);
+    var day = msgDate.getDate();
+    switch(day) {
+      case 0: day = "00"; break; case 1: day = "01"; break; case 2: day = "02"; break; case 3: day = "03"; break; case 4: day = "04"; break; case 5: day = "05"; break; case 6: day = "06"; break; case 7: day = "07"; break; case 8: day = "08"; break; case 9: day = "09"; break;
```

Comment:
Unsure why you wanted to make these with a leading zero.
Here's a better algo for that. Reads easier than switch and in fewer chars
`
var months = {0: "Jan", 1: "Feb", 2: "Mar"...};
var month = months[msgDate.getMonth()];
`